src/routes/api/profile.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import ProfileController from '../../controllers/profileController';
+import middleware from '../../middlewares';
+
+const { Authenticate, validate } = middleware;
+
+const router = Router();
+
+router.route('/')
+  .post(Authenticate.verifyToken, ProfileController.upsertProfile)
+  .get(Authenticate.verifyToken, Authenticate.verifyTravelAdmin, ProfileController.getAllProfiles);
```

Comment:
I am actually targeting the super admin in that query. Sama is looking for whether the role is either Travel admin or super admin. I had a separate method for it as part of my own class before now. I will have to go and create it again, so that I don't give Travel admin the same authority as super admin.

Reply:
Like I said, in the class I wrote originally, I gave it to the super admin. This was an omission of not considering that the travel admin will get the same privilege. An omission I will rectify now.

Reply:
Okay. Effecting all that right away.

Reply:
Done.